SonarScanner.Shim/Resources.Designer.cs

```diff
@@ -1,10 +1,10 @@
 ﻿//------------------------------------------------------------------------------
 // <auto-generated>
-//     This code was generated by a tool.
-//     Runtime Version:4.0.30319.42000
+//     Ce code a été généré par un outil.
```

Comment:
I removed the French Language Pack and installed English one on VS. Fixed the issue.